src/seedsigner/views/psbt_views.py

```diff
@@ -79,13 +79,6 @@ def run(self):
             return Destination(SeedMnemonicEntryView)
 
         elif button_data[selected_menu_num] == self.TYPE_ELECTRUM:
-            self.run_screen(
-                    WarningScreen,
-                    title="Electrum warning",
-                    status_headline=None,
-                    text=f"Some features disabled for Electrum seeds",
-                    show_back_button=False,
-            )
             from seedsigner.views.seed_views import SeedMnemonicEntryView
             self.controller.storage.init_pending_mnemonic(num_words=12, is_electrum=True)
             return Destination(SeedMnemonicEntryView)
```

Comment:
I think you meant this to direct to the new SeedElectrumMnemonicStartView since removed the warning screen above